python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -95,9 +97,20 @@ class DiracBase(IBackend):
     _hidden = True
 
     def _setup_subjob_dataset(self, dataset):
+        """
+        Unknown what this ethod does other than return None:
+        Args:
+            Dataset (Dataset): This is a GangaDataset object, todo check this isn't a list
+        """
         return None
 
     def _setup_bulk_subjobs(self, dirac_ids, dirac_script):
+        """
+        This is the old bulk submit method which is used to construct the subjobs for a paraetric job
```

Comment:
We don't properly support the bulk submit using the parametric stuff and haven't supported it for a long time now as far as I know. It may work on the GridPP dirac without problems but it caused issues for LHCb